testnet/testnet-rewards.md

```diff
@@ -69,14 +69,124 @@ Validators earn $JOY allocations in the same way as for bounty contributions: a
 
 ### Content Creator
 
 `<Coming>`
+
+
+## Scoring and Reward Example
+To clarify how the scoring and rewards works, here is straightforward example.
+Note that the inputs are somewhat random, so don't set your expectations based on them.
+
+### Council Period Parameters and weights
+| Name                  | Value                     |
+| --------------------- |:-------------------------:|
+| `JOY_BUDGET`          | 2,000,000/0.2%/USD120,000 |
+| `USD_SUBSIDY`         | 2,000                     |
+| `tJOY_BUDGET`         | 80,000,000                |
+| `COUNCIL_tJOY_REWARD` | 10,000,00                 |
+
+| Group                 | Weight                    |
+| --------------------- |:-------------------------:|
+| `B_W`                 | 5                         |
+| `C_W`                 | 3                         |
+| `D_W`                 | 6                         |
+| `F_W`                 | 1                         |
+| `HR_W`                | 5                         |
+| `M_W`                 | 2                         |
+| `P_W`                 | 2                         |
+| `SU_W`                | 2                         |
+| `LO_W`                | 1                         |
+| `SUM`                 | **27**                    |
+
+### Budget Allocation
+
+Of the tJOY 80M in the budget, we assume tJOY 78M was spent, as shown below:
+| Purpose         | Spending [tJOY] | Workers (ex Lead) | Lead [tJOY] | Workers [tJOY] |
+|-----------------|:---------------:|:-----------------:|:-----------:|:--------------:|
+| Builders        | 15              | 5                 | 3           | 2.4            |
+| Content         | 10.5            | 6                 | 1.8         | 1.45           |
+| Distributor     | 17              | 8                 | 3           | 1.75           |
+| Forum           | 3               | 2                 | 1.5         | 0.75           |
+| Human Resources | 16              | 20                | 2           | 0.7            |
+| Marketing       | 7               | 3                 | 1           | 2              |
+| Council         | 6               | 5                 | NA          | 1.2            |
+| `SUM`           | **74.5**        | **49**            | **12.3**    | **62.2**       |
+
+### WG Rewards
+As the WG rewards are not (directly\*) impacted by their performance, we get the following rewards for the Workers and Leads:
+
+| Purpose         | Spending [MtJOY] | Workers (ex Lead) | Workers [MtJOY] | Workers [JOY] | Workers [USD]   | Lead [MtJOY] | Lead [JOY] | Lead [USD]      |
+|-----------------|:----------------:|:-----------------:|:---------------:|:-------------:|:---------------:|:------------:|:----------:|:---------------:|
+| Builders        | 15               | 5                 | 2.4             | 60000         | 60 - 3600       | 3            | 75000      | 75 - 4500       |
```

Comment:
JOY number is super misleading, make it a %
even if it's 0.0005%

Reply:
Keeping for now...